docs/qna.md

```diff
@@ -16,6 +16,8 @@ There are several reasons:
 1. According to SmartBear, [Swagger Codegen 2.x and 3.x should be supported in parallel for a while](https://github.com/swagger-api/swagger-codegen/issues/7754#issuecomment-375039048) without the possibility to work with git branches to merge the fixes from one branch to the next. Having to implement everything twice is not a good idea and the best use of the Swagger Codegen community resources.
 1. Having a community-driven version can bring the project to the next level.
 
+UPDATE: After the public launch of OpenAPI Generator, some of the core team members were contacted by Ron (SmartBear) and we explained to him the reasons of the fork as explained above. We also asked him to contact team@openapitools.org if he wants to discuss further but so far we don't see any email from him to team@openapitools.org.
```

Comment:
I would remove "we":

> and explained to him the reasons of the fork as explained above.

And later:

> He was asked to contact team@openapitools.org for further discussion, but so far there was no email from him to team@openapitools.org

Reply:
"we" is appropriate here, as William and I both had separate interactions with him. In British English, there's also a "Royal We" which is used by single people of office or status; it can be used more generally to distance a single person or entity from a damaging situation. In both cases, "we" would be used correctly in this document.

Reply:
I had contact with Ron too... I can provide Gitter logs...

It is just the rest of the page written in third person and for this section there is switch to "we". I style problem in my opinion...

But it is a detail.